### PR TITLE
[Eduard Hrabchak] fetch api

### PR DESCRIPTION
**Pull request by EdGrLab**

### CODE REVIEW

File: css-practice/task3/js/blog-section.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/* global fetch, document */
+const latestBlogs = document.querySelector('.stories-container');
```
Comment (dverbovyi) [inline css-practice/task3/js/blog-section.js]: global scope variable detected!

**Comment by dverbovyi** [comment]
@EdGrLab please resolve conflicts

**Comment by Hollister009** [comment]
Done

File: css-practice/task3/js/fetch-api.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+function dateParser(dateString) {
+  const date = new Date(dateString);
+  return {
+    day: date.getDate(),
+    month: date.toLocaleString('en-us', {month: 'short'}),
+    year: date.getFullYear(),
+  };
+}
+
+function renderBlog(blog) {
```
Comment (dverbovyi) [inline css-practice/task3/js/fetch-api.js]: a function name is not appropriate, rename to prepareBlogModel

**Pull request merged**